src/app/components/BCP47Tag.svelte

```diff
@@ -14,8 +14,19 @@
   }
 </script>
 
+<style>
+  :global().input-field__label-bcp {
```

Comment:
I am SUPER CONFUSED about this selector.

Reply:
the `.input-field__label-bcp` should be inside the global()